<launch>
	<!-- main node -->
	<arg name="config_file"/>
	
	<node pkg="epic_planner" name="exploration_node" type="exploration_node"
		output="screen">
		<rosparam file="$(find epic_planner)/config/$(arg config_file)" command="load" />
		<!-- Fsm -->
		<param name="fsm/thresh_replan" value="0.5" type="double" />
		<param name="fsm/emergency_replan_control_error" value="1.5" type="double" />
		<param name="fsm/replan_time" value="0.2" type="double" />
		<param name="fsm/debug_planner" value="false" type="bool" />
		<!-- Exploration manager -->
		<param name="exploration/tsp_dir" value="$(find lkh_tsp_solver)/resource" type="string" />
		<param name="bubble_astar/allocate_num" value="1000000" type="int" />
		<param name="bubble_astar/lambda_heu" value="100.0" type="double" />
		<param name="bubble_astar/debug" value="false" type="bool" />
		<param name="bubble_astar/resolution_astar" value="0.2" type="double" />
		<!-- topomap -->
		<param name="bubble_topo/cube_discrete_size" value="0.3" type="double" />
	</node>
</launch>